barretenberg/cpp/scripts/analyze_vm_compile_time.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -eu
+# So we can glob recursively
+shopt -s globstar
+
+PRESET="${1:-wasm-threads}"
```

Comment:
Is there any reason we default to wasm?

Reply:
This is mostly for compatibility with the `analyze_compile_time.sh` script in case we need to call it (if a caller hasnt before)